app/components/login.jsx

```diff
@@ -1,202 +1,318 @@
+                    onChange={e => methods.onChangeConfirm(e.target.value)}
+                />
+                <div className={classes.agreeTermContainer}>
+                    <div className={classes.checkTerm}>
+                        <input
+                            className={classes.checkTermBox}
+                            type='checkbox'
+                            name='agreed'
+                            onClick={e => methods.onChangeAgree(e.target.checked)}
+                        />
+                        <label className={classes.agreeTermLabel}>
+                            I agree to the
+                            <a href='https://enciv.org/terms' target='_blank' className={classes.aLinkTerm}>
+                                Terms of Service
+                            </a>
+                        </label>
+                    </div>
+                </div>
+                <div className={classes.btnContainer}>
+                    <button className={cx(classes.btn, isLogIn && classes.disabled)} onClick={e => methods.signup()}>
+                        Sign Up
+                    </button>
+                    <button className={cx(classes.btn, !isLogIn && classes.disabled)} onClick={e => methods.login()}>
+                        Log In
+                    </button>
+                    <button className={cx(classes.btn, !isLogIn && classes.disabled)} onClick={e => methods.skip()}>
+                        Skip
+                    </button>
+                </div>
+                <div className={cx(classes.resetPasswordBtn, !isLogIn && classes.disabled)}>
+                    <button onClick={e => methods.sendResetPassword()} className={classes.resetBtn}>
+                        Send Reset Password
+                    </button>
+                </div>
+                <div className={cx("go-to-sign-up", isLogIn && classes.disabled)}>
+                    <p> Don't have an account? Click Join, it only takes a minute to sign up!</p>
+                </div>
+                <div>
+                    {state.error && <div style={{ color: '#fec215', textAlign: 'center' }}>{state.error}</div>}
+                    {state.info && <div style={{ color: '#fec215', textAlign: 'center' }}>{state.info}</div>}
+                    {state.success && <div style={{ color: '#fec215', textAlign: 'center' }}>{state.success}</div>}
+                </div>
+            </div>
+        </div>)
 }
 
-export default Login;
+export default React.forwardRef(Login);
+
+const useStyles = createUseStyles(theme => ({
+    Login: {
+        backgroundColor: 'white', 
+        width: '60rem',
+        maxWidth: '100vw',
+        margin: 0,
+        borderRadius: '1rem',
+        height: '100%',
+        padding: '0',
+        fontFamily: 'Inter',
+        fontSize: '16px', // change to rem?
+        fontWeight: '700',
+        lineHeight: '16px', // change to rem?
```

Comment:
never user px. always convert to rem.  Ever for borders and border shadow below.  This just makes it look the same on different devices that have different font sizes.  Desktops are mostly 16, mobile is more like 8, some are 12.